### PR TITLE
devops: Dockerfile+Makefile for all & Prettier for prime

### PR DESCRIPTION
**Pull request by novusnota**
Mini chores:
- Added Prettier and ThePrimeagens' usual config to `typescript/package.json`
- Added myself to `CODEOWNERS`, plus a couple of TODOs for it
- Re-written `pr-worthy.yml` — workflows prior to this change will fail
- Moved root `.gitignore` into the `rust/`, `zig/`, `typescript/` folders back. (prime's git may have remembered to ignore some things, but it would be nice to also make it work for everyone, who'd like to test his implementations on their own)

Big update in terms of lines, small update in terms of actual new stuff:
- Added stub Dockerfile and Makefile to all languages currently present in the repo. Existing ones weren't touched, except for:
    - Fortran: copied into repl.Dockerfile, as it's execution won't stop automatically
    - Bash: updated Dockerfile and Makefile to the newest revision.

P.S.: Workflow below failed because of too many builds and requests to Ubuntu servers with `apt-get update` and stuff. But all was good from Bash up to Elixir.

### CODE REVIEW

**Comment by novusnota** [comment]
Latest workflow failed because of too many builds and requests to Ubuntu servers with `apt-get update` and stuff. But all was good from Bash up to Elixir.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+/crystal/         @devnote-dev
+/csharp/          @MarioGK
+/dart/            @shreyassanthu77
+# TODO: Consider uniting those two:
+/elixir-shyryan/  @ryanwinchester
+/elixir/          @ryanwinchester
+/erlang/          @mikimaine
+/fortran/         @greuben92
+/fsharp/          @meenzen
+/java/            @xNaCly
```
Comment (Crain-32) [inline CODEOWNERS:20]: From my understanding on Stream yesterday I'm supposed to be a Java Code Owner, Prime could confirm.

Comment (novusnota) [reply CODEOWNERS:20]: There's a whole list of new people from last stream to put in this file, but yeah

**Comment by novusnota** [comment]
Needed a quick resolution of conflicts here after ThePrimeagen updated the global .gitignore file, eh

**Pull request closed without merging**